### DISCUSSION TITLE
A few Suggetions

### DISCUSSION THREAD

**Discussion by FuckingToasters** [diced/zipline#504] [Ideas]
Hello, first of all i want to thank you for this project, i really like it but i also think that a few additional things should be added:

1. Custom Domains
- Let Users add their own domains via DNS Records

2. Homepage
- Add a Homepage when visiting the main url (currently the homepage is set to login form)

3. Upgrades
- Implement a feature so users can pay via crypto currency for additional features.

**Comment by diced** [comment]
1. u can just do it with existing features
2. maybe no, i dont want to add this
3. what? no.
